test/e2e/framework.go

```diff
@@ -969,19 +970,20 @@ func forAllNodes(fn func(nodeName string) error) error {
 	return nil
 }
 
-// forAllAntreaPods invokes the provided function for every Antrea Pod currently running on every Node.
-func (data *TestData) forAllAntreaPods(fn func(nodeName, podName string) error) error {
+// forAllNsPods invokes the provided function for every Pod currently running on every Node in a given namespace and
+// matches labelSelector criteria.
```

Comment:
and *which* matches *the* labelSelector criteria

Reply:
Done